pkg/debug/scripts/collect-info.sh

```diff
@@ -64,9 +66,20 @@ while getopts "vhsa:dj" o; do
         a)
             READ_LOGS_APP="$OPTARG"
             ;;
+        t)
+            READ_LOGS_DAYS="$OPTARG"
+            if [ "$READ_LOGS_DAYS" -lt 1 ] || [ "$READ_LOGS_DAYS" -gt 30 ]; then
+                echo "Error: READ_LOGS_DAYS must be between 1 and 30."
+                exit 1
+            fi
+            ;;
         d)
             READ_LOGS_DEV=1
             ;;
+        e)
+            INFO_DIR_SUFFIX="eve-info-edgeview-v$VERSION-$DATE"
+            TARBALL_FILE="/persist/eve-info/$INFO_DIR_SUFFIX.tar.gz"
```

Comment:
How about moving this duplicated line after the getopts loop.

And it is a bit more readable if the default INFO_DIR_SUFFIX is set just above the getopts loop.